07-routing-gossip.md

```diff
@@ -0,0 +1,113 @@
+# BOLT #7: P2P Node and Channel Discovery
+
+This specification describes a simple initial node and channel discovery mechanism that does not rely on a third-party to disseminate the information.
+
+Node and channel discovery serve two different purposes:
+
+ - Channel discovery allows the creation and maintenance of a local view of the network's topology such that the node can discover routes to desired destination.
+ - Node discovery allows nodes to broadcast their ID, host and port, such that other nodes can open connections and establish payment channels.
+ 
+Peers in the network exchange `route_announcement` messages that contain information about a node and about its outgoing channels.
+The node creating an announcement is referred to as the _announcing node_.
+Announcements MUST have at least one associated channel whose existence can be proven by inspection of the blockchain, i.e., the anchor transaction creating the channel MUST have been confirmed.
```

Comment:
6 confirmations seems fine. My point was that if we just wait for one confirmation (which is what the spec seems to be saying) we might have issues with some nodes not having received the new block yet.